dev/tabs.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+        <!--style>
+            .title-tab {display:block;text-decoration:underline;padding-right:5px}
+            .bdy  {display:none;border:none;padding:14px 16px;}
+            .tab-active {display:block !important;background-color: #ccc;}
+            .tab1 > .bdy {display:none}
+            button {display:inline}
+        </style-->
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>Tabs Demo</title>
+        <script type="module" src="../src/tabs.ts"></script>
+    </head>
+    <body>
+        <h1>Tabs Demo</h1>
+        <script>
+        </script>
+        <div>
+            <aalam-tabs class="tab1" colsize="title:40%;body:60%" animation="b2t" fashion="xs:row;l:column">
+                <div id="tabslot" slot="tab-title" class="title-tab">title1</div>
+                <div class="bdy" slot="tab-body">body1</div>
+
+                <div slot="tab-title" class="title-tab tab-active">title2</div>
+                <div slot="tab-body" class="bdy">body2</div>
+
+                <div slot="tab-title" class="title-tab tab-active">title3</div>
+                <div slot="tab-body" class="bdy">body3</div>
+
+            </aalam-tabs>
+
+            <aalam-tabs class="tab1" animation="show:r2l;hide:l2r" fashion="xs:column;s:row;l:column;xl:accordion">
+                <button slot="tab-title" class="title-tab">title 1</button>
+                <div class="bdy" slot="tab-body">body 1</div>
+                
+                <button slot="tab-title" class="tab-active title-tab">title 2</button>
+                <div class="bdy" slot="tab-body">body 2</div>
+                
+                <button slot="tab-title" class="title-tab tab-active">title 3</button>
+                <div class="bdy" slot="tab-body">body 3</div>
+            </aalam-tabs>
```

Comment:
It could be more neat. For example, if its column, we can have a border. Better to have some between the two demo tabs